Makefile

```diff
@@ -65,12 +65,23 @@ fill-cache-for-e2e-regression: oasis-indexer
 
 # Run the api tests locally, assuming the environment is set up with an oasis-node that is 
 # accessible as specified in the config file.
-test-e2e-regression: export TZ=UTC
 test-e2e-regression: oasis-indexer
-	./oasis-indexer --config tests/e2e_regression/e2e_config.yml analyze
+	TZ=UTC ./oasis-indexer --config tests/e2e_regression/e2e_config.yml analyze
 	@$(ECHO) "$(CYAN)*** Indexer finished; starting api tests...$(OFF)"
 	./tests/e2e_regression/run.sh
 
+# Accept the outputs of the e2e tests as the new expected outputs.
+accept-e2e-regression:
```

Comment:
add to .PHONY?

Reply:
:shrug: We don't really use/need it. Added, and half-heartedly added several other e2e-related targets that were missing, but didn't make a careful pass to ensure _every_ relevant target is listed under .PHONY.